.storybook/primitives-v8.css

```diff
@@ -10,12 +10,14 @@
 @import '@primer/primitives/tokens-next-private/css/functional/typography/typography.css';
 
 /* color */
-@import '@primer/primitives/tokens-next-private/css/functional/themes/light.css';
-@import '@primer/primitives/tokens-next-private/css/functional/themes/light-tritanopia.css';
-@import '@primer/primitives/tokens-next-private/css/functional/themes/light-high-contrast.css';
-@import '@primer/primitives/tokens-next-private/css/functional/themes/light-colorblind.css';
-@import '@primer/primitives/tokens-next-private/css/functional/themes/dark.css';
-@import '@primer/primitives/tokens-next-private/css/functional/themes/dark-colorblind.css';
-@import '@primer/primitives/tokens-next-private/css/functional/themes/dark-dimmed.css';
-@import '@primer/primitives/tokens-next-private/css/functional/themes/dark-high-contrast.css';
-@import '@primer/primitives/tokens-next-private/css/functional/themes/dark-tritanopia.css';
+/* temporarily disabiling these to ensure visual regression tests work on production code
```

Comment:
do we want to keep these disabled in storybook?

Reply:
I'd like to add a toolbar addon to let you toggle them on, like how we have it in Primer View Components Lookbook

<img width="331" alt="primitives toggle" src="https://github.com/primer/react/assets/18661030/a924bfe9-881c-4f93-a4c2-2dc201e4d764">

Reply:
Good idea, should we do that in a different PR though and get this one quickly?

Reply:
Yeah totally agree